artemis-protocols/artemis-openwire-protocol/src/main/java/org/apache/activemq/artemis/core/protocol/openwire/OpenWireConnection.java

```diff
@@ -742,6 +756,25 @@ public void addConsumer(ConsumerInfo info) throws Exception {
       }
    }
 
+   public void setConnectionEntry(ConnectionEntry connectionEntry) {
+      this.connectionEntry = connectionEntry;
+   }
+
+   public void setUpTtl(final long inactivityDuration, final long inactivityDurationInitialDelay, final boolean useKeepAlive) {
+      this.useKeepAlive = useKeepAlive;
+      this.maxInactivityDuration = inactivityDuration;
+
+      protocolManager.getScheduledPool().schedule(new Runnable() {
```

Comment:
Say the client->server ping is 1 minute.
The server->client ping is configured to be 1 seconds.

Shouldn't the server ping the client every 1 second in case of inactivity?